packages/react-scripts/scripts/start.js

```diff
@@ -258,6 +259,31 @@ function run(port) {
   runDevServer(host, port, protocol);
 }
 
+function isProcessAReactApp(processCommand) {
+  return /^node .*react-scripts\/scripts\/start\.js\s?$/.test(processCommand);
+}
+
+function getProcessForPort(port) {
```

Comment:
Let's move this into `react-dev-utils` package so that every user who ejected doesn't need to inherit this code.